src/commands/opsGenieCommands.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import got from 'got'
+
+export async function createOpsGenieAlert(message: string, alias: string, apiKey: string, accountId: string) {
+  const options = {
+    json: {
+      message: message,
+      priority: 'P5',
+      alias: alias,
+      details: { AWSAccountId: accountId }
+    },
+    headers: { Authorization: `GenieKey ${apiKey}` }
+  }
+
+  const URL = 'https://api.eu.opsgenie.com/v2/alerts/'
+  const data: any = await got.post(URL, options).json()
+
+  return data
+}
+
+export async function getOpsGenieAlert(alias: string, apiKey: string) {
+  const options = {
+    headers: {
+      Authorization: `GenieKey ${apiKey}`
+    }
+  }
```

Comment:
Pull the common parameters up to re-use?

```suggestion
function authorizationHeaders(apiKey: string) {
  return {
    headers: {
      Authorization: `GenieKey ${apiKey}`
    }
  }
  
export async function getOpsGenieAlert(alias: string, apiKey: string) {
  const options = {
    ...authorizationHeaders(apiKey),
  }
```

They could then be referenced elsewhere